css/site.styl

```diff
@@ -107,13 +114,29 @@ img
     opacity: 1
   100% 
     opacity: 1
-  
-input
-  height: 0
-  width: 0
-  opacity: 0
-  text-transform: uppercase
 
+// CONVERSATION FORM
+.conversation
+
+  #custom-form
```

Comment:
cuz that class change… i actually really dislike using id's for styling but i dont wanna go there now